bin/command.php

```diff
@@ -561,7 +564,11 @@ private static function parse_docblock( $docblock ) {
 		}
 		$ret['description'] = str_replace( '\/', '/', trim( $ret['description'], PHP_EOL ) );
 		$bits = explode( PHP_EOL, $ret['description'] );
-		$ret['short_description'] = array_shift( $bits );
+		$short_desc = array( array_shift( $bits ) );
+		if ( !empty( $bits[0] ) ) {
+			$short_desc[] = array_shift( $bits );
+		}
+		$ret['short_description'] = trim( implode( PHP_EOL, $short_desc ), PHP_EOL );
```

Comment:
I would implode with spaces here, not newlines. I don't think it makes sense to keep newlines around for short descriptions.